.github/workflows/rust-audit.yml

```
@@ -0,0 +1,35 @@
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      
+      - name: Install cargo-audit
+        run: cargo install cargo-audit
+      
+      - name: Run security audit
+        uses: actions-rs/cargo@v1
+        with:
+          command: audit
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)